guides/source/initialization.md

```diff
@@ -158,19 +159,20 @@ defined here to find the matching command.
 
 ### `rails/commands/command_tasks.rb`
 
-When one types an incorrect rails command, the `run_command` is responsible for
-throwing an error message. If the command is valid, a method of the same name
-is called.
+If the command is part of the COMMAND_WHITELIST, a method of the same name is called,
+if not we proxy it to rake.
```

Comment:
I'd rather keep the old comment, we already copy the implementation, so we don't need to add more implementation details.